src/RedLock/RedLock.rn.md

```diff
@@ -1,3 +1,8 @@
+### 4.0.1 - xx.xx.2019
```

Comment:
@alsami Could you please set some date here, so we can merge this?

Reply:
Done!